src/app/portlets/content-types/layout/content-types-layout.component.html

```diff
@@ -13,14 +13,20 @@ <h4 class="content-type__fields-sidebar-title">{{ i18nMessages['contenttypes.sid
                 </div>
             </div>
         </p-tabPanel>
-        <p-tabPanel class="content-type__relationships" header="{{ i18nMessages['contenttypes.tab.relationship.header'] }}" *ngIf="contentTypeId">
-            <dot-iframe [src]="relationshipURL"></dot-iframe>
+        <p-tabPanel class="content-type__relationships" header="{{ i18nMessages['contenttypes.tab.relationship.header'] }}" *ngIf="contentTypeId" [cache]="false">
+            <ng-template pTemplate="content">
+                <dot-iframe [src]="relationshipURL"></dot-iframe>
+            </ng-template>
         </p-tabPanel>
-        <p-tabPanel class="content-type__permissions" header="{{ i18nMessages['contenttypes.tab.permissions.header'] }}" *ngIf="contentTypeId">
-            <dot-iframe [src]="permissionURL"></dot-iframe>
+        <p-tabPanel class="content-type__permissions" header="{{ i18nMessages['contenttypes.tab.permissions.header'] }}" *ngIf="contentTypeId" [cache]="false">
+            <ng-template pTemplate="content">
+                <dot-iframe [src]="permissionURL"></dot-iframe>
+            </ng-template>
         </p-tabPanel>
-        <p-tabPanel class="content-type__push_history" header="{{ i18nMessages['contenttypes.tab.publisher.push.history.header'] }}" *ngIf="contentTypeId">
-            <dot-iframe [src]="pushHistoryURL"></dot-iframe>
+        <p-tabPanel class="content-type__push_history" header="{{ i18nMessages['contenttypes.tab.publisher.push.history.header'] }}" *ngIf="contentTypeId" [cache]="false">
+            <ng-template pTemplate="content">
+                <dot-iframe [src]="pushHistoryURL"></dot-iframe>
+            </ng-template>
         </p-tabPanel>
```

Comment:
can do we test this change anyway?

Reply:
Is tested... in this [line](https://github.com/dotCMS/core-web/pull/585/files#diff-d8cd5b4e79e98bac770f380cd31252a8R162) you can see that now I have to set the ```selected``` in true in order to get the iframe, if not the iframe is null.